compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/schema/generator/SchemaGenerator.java

```diff
@@ -209,7 +209,7 @@ private Field getField(ResourceMethodSymbol methodSymbol) {
     private Field getField(ResourceMethodSymbol methodSymbol, List<PathSegment> list) {
         if (list.size() == 1) {
             boolean isDeprecated = methodSymbol.deprecated();
```

Comment:
Are we using this variable elsewhere ? otherwise can pass this inline. Shall we check for similar places ?

Reply:
It uses twice inside this function and no harm of initializing a local variable for that right?